src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -355,4 +358,64 @@ public without sharing class UTIL_CustomSettingsFacade {
             }
         }
     }
+
+    /*******************************************************************************************************
+    * @description Gets if Course Connections is enabled
+    * @return Boolean
+    */
+    public static Boolean courseConnectionsEnabled() {
+        if (!Test.isRunningTest()) {
```

Comment:
Not a huge fan of this if statement that checks if we are inside a test. Any other way to work around this?